dpe/src/commands/rotate_context.rs

```diff
@@ -47,7 +55,16 @@ impl<C: Crypto> CommandExecution<C> for RotateCtxCmd {
 
         // Make sure the command is coming from the right locality.
         if dpe.contexts[idx].locality != locality {
-            return Err(DpeErrorCode::InvalidHandle);
+            return Err(DpeErrorCode::InvalidLocality);
+        }
+        
+        // Make sure caller's locality does not already have a default context.
+        if self.uses_target_is_default() {
+            let default_context_idx =
+                dpe.get_active_context_pos(&ContextHandle::default(), locality);
+            if default_context_idx.is_some() {
+                return Err(DpeErrorCode::InvalidArgument);   
+            }
         }
 
         let new_handle = dpe.generate_new_handle()?;
```

Comment:
Not sure we want it as part of this PR, but if the request is to make it default, it should assign the `new_handle` to be all zeroes (which indicates a default context).  We should probably add a unit test for this as well.
```suggestion
        let new_handle = if self.uses_target_is_default() {
            dpe.generate_new_handle()?
        } else {
            ContextHandle::default()
        };
```

Reply:
Thanks, will do in followup commit